Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/AppCmdOnTargetMachines.ps1

```diff
@@ -168,7 +168,7 @@ function Enable-SNI
         $ipAddress = "*"
     }
 
-    $appCmdArgs = [string]::Format(' set site /site.name:{0} /bindings.[protocol=''https'',bindingInformation=''{1}:{2}:{3}''].sslFlags:"1"',$siteName, $ipAddress, $port, $hostname)
+    $appCmdArgs = [string]::Format(' set site /site.name:"{0}" /bindings.[protocol=''https'',bindingInformation=''{1}:{2}:{3}''].sslFlags:"1"',$siteName, $ipAddress, $port, $hostname)
     $command = "`"$appCmdPath`" $appCmdArgs"
```

Comment:
Can you increase patch version and update change log 1.2.4 release section so that we can publish this fix as part of that